package-lock.json

```diff
@@ -16398,6 +16398,11 @@
 					"version": "0.5.2",
 					"resolved": "https://registry.npmjs.org/type-fest/-/type-fest-0.5.2.tgz",
 					"integrity": "sha512-DWkS49EQKVX//Tbupb9TFa19c7+MK1XmzkrZUR8TAktmE/DizXoaoJV6TZ/tSIPXipqNiRI6CyAe7x69Jb6RSw=="
+				},
```

Comment:
Not sure why this was removed in #2146, but when I `npm install`ed, the lockfile updated.